File: Algoritma/Cek Bilangan Prima/Cek-Bilangan-Prima.py
batas = int (input ())

for i in range (batas):
    angka = int (input ())
    prima = 0

    if angka == 1:
        print ("Bukan bilangan prima")
        continue

    for j in range (2, int (angka ** 0.5) + 1):
        if angka % j == 0:
            prima += 1
            break

    print ("Bilangan prima" if prima == 0 else "Bukan bilangan prima")

File: Algoritma/Cek Bilangan Prima/CekBilanganPrima.java
import java.util.Scanner;

public class BilanganPrima
{
    public static void main(String [] args)
    {
        Scanner scan = new Scanner (System.in);
        int batas = scan.nextInt ();

        for (int i = 0; i < batas; i++)
        {
            int angka = scan.nextInt ();
            int prima = 0;

            if (angka == 1)
            {
                System.out.println ("Bukan bilangan prima");
                continue;
            }

            for (int j = 2; j * j <= angka; j++)
            {
                if (angka % j == 0)
                {
                    prima++;
                    break;
                }
            }

            System.out.println (prima == 0 ? "Bilangan prima" : "Bukan bilangan prima");
        }
    }
}
